Remove `fast/media/invalid-media-query-list.html` in favor of WPT tests

iidmsa opened the pull request:
#### b1c91998045c7184cf0e78ed5dce8fe859920df5
<pre>
Remove `fast/media/invalid-media-query-list.html` in favor of WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=294424">https://bugs.webkit.org/show_bug.cgi?id=294424</a>
<a href="https://rdar.apple.com/problem/153253490">rdar://problem/153253490</a>

Reviewed by Anne van Kesteren.

This patch removes local copy of `fast/media/invalid-media-query-list.html` test
since it is covered by below WPT tests:

&gt; mediaqueries/mq-invalid-media-type-001.html
&gt; mediaqueries/mq-invalid-media-type-002.html
&gt; mediaqueries/mq-invalid-media-type-003.html
&gt; mediaqueries/mq-invalid-media-type-004.html
&gt; mediaqueries/mq-invalid-media-type-005.html
&gt; mediaqueries/mq-invalid-media-type-layer-001.html
&gt; mediaqueries/mq-invalid-media-type-layer-002.html

Relevant tests were synced in 252744@main.

* LayoutTests/fast/media/invalid-media-query-list-expected.html: Removed.
* LayoutTests/fast/media/invalid-media-query-list.html: Removed.
* LayoutTests/platform/ipad/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/296227@main">https://commits.webkit.org/296227@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1f36aeb186092e16da6d343cd0eefab3fb2952ad

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/107597 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/27280 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/17691 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/112825 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/58139 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/27972 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/35782 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/81705 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/110526 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/22168 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/96984 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/62085 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/21604 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/15123 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/57578 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/91547 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/15157 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/115915 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/34665 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/25598 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/90734 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/35041 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/93234 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/90475 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/23108 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/35387 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/13173 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/30428 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/34586 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/34332 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/37692 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/35995 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov reviewed (commented):
I'm not sure if we should be doing this. `git log` for this file shows history and motivation for this test; WPT tests bear no history or importance.

This is kind of a muted case, as our history for this test is not really revealing very much, but it's still better than WPT.

annevk reviewed (approved):
I don't really see the value of history in this case (there's a solid specification for this behavior and a test suite). And we also have to consider ongoing maintenance and CI not being free. Deduplicating is worth doing.

iidmsa committed:
Remove `fast/media/invalid-media-query-list.html` in favor of WPT tests
https://bugs.webkit.org/show_bug.cgi?id=294424
rdar://problem/153253490

Reviewed by Anne van Kesteren.

This patch removes local copy of `fast/media/invalid-media-query-list.html` test
since it is covered by below WPT tests:

> mediaqueries/mq-invalid-media-type-001.html
> mediaqueries/mq-invalid-media-type-002.html
> mediaqueries/mq-invalid-media-type-003.html
> mediaqueries/mq-invalid-media-type-004.html
> mediaqueries/mq-invalid-media-type-005.html
> mediaqueries/mq-invalid-media-type-layer-001.html
> mediaqueries/mq-invalid-media-type-layer-002.html

Relevant tests were synced in 252744@main.

* LayoutTests/fast/media/invalid-media-query-list-expected.html: Removed.
* LayoutTests/fast/media/invalid-media-query-list.html: Removed.
* LayoutTests/platform/ipad/TestExpectations:

Canonical link: https://commits.webkit.org/296227@main
